data/ovs/002-ovs-daemonset.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ovs-cni-amd64
+  namespace: ovs
+  labels:
+    tier: node
+    app: ovs-cni
+spec:
+  selector:
+    matchLabels:
+      app: ovs-cni
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: ovs-cni
+    spec:
+      serviceAccountName: ovs-cni-marker
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+{{ if .EnableSCC }}
+      initContainers:
+        - name: ovs-vsctl
+          command:
+            - /bin/bash
+            - -c
+            - |
+              cp /usr/bin/ovs-vsctl /host/usr/local/bin/_ovs-vsctl
+              echo '#!/usr/bin/bash
+              _ovs-vsctl --db unix:///run/openvswitch/db.sock $@
+              ' > /host/usr/local/bin/ovs-vsctl
+              chmod +x /host/usr/local/bin/ovs-vsctl
+          image: docker.io/openshift/origin-node:v3.10.0-rc.0
```

Comment:
We should parameterize this

Reply:
This PR is waiting for this one first https://github.com/kubevirt/ovs-cni/pull/75

Then we can use the same image also here because the ovs container will contain the ovs-vsctl binary

Reply:
ack